hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -153,7 +153,7 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
     val ignoredDefns: Set[BlockMemberSymbol] = Set.empty,
     val inScopeDefns: Map[BlockMemberSymbol, Set[BlockMemberSymbol]] = Map.empty,
     val modLocals: Map[BlockMemberSymbol, Local] = Map.empty,
-    val localCaptureSyms: Map[Local, (VarSymbol, BlockMemberSymbol)] = Map.empty,
```

Comment:
It seems the `BlockMemberSymbol` info was redundant.

Reply:
I guess maybe we want to the funny comment about the `BlockMemberSymbol` there for `localCaptureSyms`, which was added by me 😬